mem-08-m.xml

```diff
@@ -7604,8 +7604,8 @@ See under {wot:n} for the verb suffixes.</column>
       <column name="synonyms"></column>
       <column name="antonyms"></column>
       <column name="see_also">{qarpal:n}, {loS reD mey':n}</column>
```

Comment:
I'm not particularly consistent about this myself, but when I try to make it so there's only one link to another entry per entry. (If multiple references to another entry are necessary, they are usually marked "nolink".) So {qarpal:n} can be removed from the "see_also" if it's directly referenced in the "notes".

But also, when you edit the English note, please let me know so I can retranslate the other languages. (I might miss these if they're part of a larger change so it would be better if you let me know directly.)

Reply:
Okay, I'll tell you next time.